core/src/main/java/nl/nn/adapterframework/doc/ElementGroupManager.java

```diff
@@ -0,0 +1,98 @@
+package nl.nn.adapterframework.doc;
```

Comment:
Vergeet de license informatie niet :)

Reply:
Gedaan. Maar ik heb niet gekeken of ik "2020, 2021" of alleen "2021" moet neerzetten.